### PR TITLE
chore: add luca kündig as maintainer

### PR DESCRIPTION
**Pull request by oliverbaehler**
Hello Maintainers.

This is my request to add my good friend @lucakuendig as an official maintainer for Capsule. We have worked together at Bedag and introduced Capsule to Bedag. Therefor he has strong knowledge we can profit from.

Another aspect of this action is to take pressure from @prometherion and @bsctl. They are currently focused on Kamaji and their business and I don't want to bother them to much with pull request. Building a larger community will require more maintainers. And I can assure, he's a fun pal!

Also note, he's currently employed by Swisscom, the largest Telecom provider in Switzerland.

### CODE REVIEW

**Review by bsctl — COMMENTED** [review]
LGTM

**Comment by maxgio92** [comment]
Thank you @oliverbaehler, I agree that it would be very beneficial for the project to have further maintainers onboard, especially with respect to the maintainers affiliation diversity.

Nevertheleess due to the duties and privileges of a maintainer has with respect to a project, which are mainly in my opinion:
- lead the project goals and roadmap, influencing the steering
- review and approve contributions
to be honest it sounds a bit strange and risky for a project, to onboard new maintainers that will have the duties and privileges mentioned above, without any contribution at all, especially for a project not just created from scratch.

Don't get me wrong, it's good and fine to sponsor maintainers, and I thank you for that.

TL;DR any contribution is more than welcome, but as a maintainer would have steering responsibilities I'd like to first see contributions that also prove knowledge of a project.

My 2 cents

**Comment by oliverbaehler** [comment]
@lucakuendig feel free to reopen if you are still interested.
